### ISSUE TITLE
StoryBook: Improve navigation

### ISSUE THREAD

**Issue by manuq** [endlessm/threadbare#2339]
Make it similar to the Pause overlay:
- Use a TabContainer and make each page a tab. Use "previous_tab", "next_tab" input actions for navigating the pages. Althogh this one should have "tabs visible" disabled.

Done:
- Have a row with input hints at the bottom.
- Remove the custom arrow buttons from the sides.

**Comment by manuq** [comment]
Here is useful information about font sizes in the StoryBook: https://github.com/endlessm/threadbare/pull/2343#discussion_r3394734427 . To consider while tweaking its user interface.
